### ISSUE TITLE
google.script not found in development mode

### ISSUE THREAD

**Issue by Raj00377** [enuchi/React-Google-Apps-Script#163]
I have configured using it `gas-client` package 

```
const { serverFunctions } = new GASClient<typeof publicServerFunctions>({
  // this is necessary for local development but will be ignored in production
  allowedDevelopmentDomains: (origin) => 
    (/https:\/\/.*\.googleusercontent\.com$/.test(origin) || origin === 'https:localhost:3000') ? true : false
  }
})
```

Anyone has faced this issue? @enuchi

**Comment by enuchi** [comment]
Should not add your localhost address there. Can you share what exact error you're seeing, and also include what platform (OS)/browser/node versions you're using?

**Comment by Raj00377** [comment]
While using `npm run start`, I'm getting origin as `localhost:3000` that's why i'hve put there.
OS : Ubuntu 20.04.5 LTS 64-bit
node version : v16.18.1

![Screenshot from 2022-12-26 15-14-27](https://user-images.githubusercontent.com/40709274/209533826-474bf87f-b0e9-4b43-b1bc-b68734f1100c.png)

**Comment by enuchi** [comment]
I need to update the Readme. You only need to have the Google domain that loads the iframe there. I see you've moved some files around. Does your issue happen with a fresh install of this project?

**Comment by Raj00377** [comment]
Yes. i've add another entry point in webpack, but the server functions are imported from the `/client/utils` only. i've been playing with this project for past 1 month. i'll try to do a fresh install.

**Comment by enuchi** [comment]
Yes, do a clean install. You shouldn't need to change `(/https:\/\/.*\.googleusercontent\.com$/.test(origin)` -- if the domain is wrong the app would still load, it just wouldn't be able to make server calls.
